appstream.patch

```diff
@@ -0,0 +1,390 @@
+Date: Fri, 19 Jul 2024 19:51:18 +0200
+Subject: [PATCH 4/9] Fix screenshot tag.
+
+---
+ dist/jumpnbump.appdata.xml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/dist/jumpnbump.appdata.xml b/dist/jumpnbump.appdata.xml
+index c0a9627..4160549 100644
+--- a/dist/jumpnbump.appdata.xml
++++ b/dist/jumpnbump.appdata.xml
+@@ -27,7 +27,7 @@
+   </content_rating>
+   <screenshots>
+     <screenshot type="default" width="800" height="512">
+-      https://gitlab.com/LibreGames/jumpnbump/raw/master/dist/screenshot.png
++      <image>https://gitlab.com/LibreGames/jumpnbump/raw/master/dist/screenshot.png</image>
+     </screenshot>
+   </screenshots>
+   <url type="homepage">https://gitlab.com/LibreGames/jumpnbump</url>
+-- 
+GitLab
+
+
+From b9c22fe8d81639d0b373200dfe17c312ed43da2d Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?= <matthias@mailaender.name>
+Date: Fri, 19 Jul 2024 20:46:36 +0200
+Subject: [PATCH 5/9] Shorten summary.
+
+---
+ dist/jumpnbump.appdata.xml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/dist/jumpnbump.appdata.xml b/dist/jumpnbump.appdata.xml
+index 4160549..0485128 100644
+--- a/dist/jumpnbump.appdata.xml
++++ b/dist/jumpnbump.appdata.xml
+@@ -8,7 +8,7 @@
+     <name>Brainchild Design</name>
+   </developer>
+   <name>Jump 'n Bump</name>
+-  <summary>Cute multiplayer platform game with exploding fluffy bunnies</summary>
++  <summary>Cute multiplayer platform game</summary>
+   <description>
+     <p>
+       Jump 'n Bump is an arcade multiplayer game for the whole family.
+-- 
+GitLab
+
+
+From 31043f3facb448eb29813837d06d3dcf86e8324b Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?= <matthias@mailaender.name>
+Date: Fri, 19 Jul 2024 20:52:32 +0200
+Subject: [PATCH 6/9] Upscale icon.
+
+---
+ Makefile           |   4 ++--
+ dist/jumpnbump.png | Bin 2819 -> 7367 bytes
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/Makefile b/Makefile
+index 077cc52..1433738 100644
+--- a/Makefile
++++ b/Makefile
+@@ -62,7 +62,7 @@ install:
+ 	mkdir -p $(DESTDIR)$(GAMEDATADIR)/jumpnbump/
+ 	mkdir -p $(DESTDIR)$(DATADIR)/metainfo/
+ 	mkdir -p $(DESTDIR)$(DATADIR)/applications/
+-	mkdir -p $(DESTDIR)$(DATADIR)/icons/
++	mkdir -p $(DESTDIR)$(DATADIR)/icons/hicolor/128x128/apps/
+ 	mkdir -p $(DESTDIR)$(DATADIR)/man/man6/
+ 	install -m 755 $(BINARIES) $(DESTDIR)$(BINDIR)/
+ 	install -m 644 data/jumpbump.dat \
+@@ -72,7 +72,7 @@ install:
+ 	install -m 644 dist/jumpnbump.desktop \
+ 		$(DESTDIR)$(DATADIR)/applications/jumpnbump.desktop
+ 	install -m 644 dist/jumpnbump.png \
+-		$(DESTDIR)$(DATADIR)/icons/jumpnbump.png
++		$(DESTDIR)$(DATADIR)/icons/hicolor/128x128/apps/jumpnbump.png
+ 	install -m 644 dist/jumpnbump.6 $(DESTDIR)$(DATADIR)/man/man6/
+ 
+ #	$(MAKE) -C menu install
+diff --git a/dist/jumpnbump.png b/dist/jumpnbump.png
+index 16ec92b9bebf96e56023e3efe21b177f2a6c2b52..a90087c0fd47914923b4503ec05cb4acdab513c8 100644
+GIT binary patch
```

Comment:
What is this doing? Changing the icon size?

Reply:
96px → 128px